packages/shared/src/components/auth/AuthOptions.tsx

```diff
@@ -63,6 +69,7 @@ export enum AuthDisplay {
 
 export interface AuthOptionsProps {
   onClose?: CloseAuthModalFunc;
+  onAuthStateUpdate?: (isLoginFlow: boolean) => void;
```

Comment:
`{ isLoginFlow }` to allow futher customization?

Reply:
Good point. Let's do that.